3rdparty/libtengine/tengine.cmake

```diff
@@ -0,0 +1,66 @@
+SET(TENGINE_VERSION "1.12.0")
+SET(OCV_TENGINE_DSTDIRECTORY ${OpenCV_BINARY_DIR}/3rdparty/libtengine)
+SET(DEFAULT_OPENCV_TENGINE_SOURCE_PATH ${OCV_TENGINE_DSTDIRECTORY}/Tengine-${TENGINE_VERSION})
+
+message ( ====DEFAULT_OPENCV_TENGINE_SOURCE_PATH = ${DEFAULT_OPENCV_TENGINE_SOURCE_PATH})
```

Comment:
Please use quotes ("") or remove this debug message.

Reply:
done